lib/routes/account.js

```diff
@@ -276,7 +277,8 @@ module.exports = function (
                   uid: account.uid,
                   kA: account.kA,
                   wrapKb: wrapKb,
-                  emailVerified: account.emailVerified
+                  emailVerified: account.emailVerified,
+                  tokenVerificationId: crypto.randomBytes(32)
```

Comment:
Hmmm...is the intention that both `sessionToken` and `keyFetchToken` should share the same `tokenVerificationId`, so that both will be verified in one go when the users clicks through the email?